test/Spec/State/Computed.spec.luau

```diff
@@ -45,7 +45,7 @@ return function()
 		local expect = getfenv().expect
 		
 		local scope = {}
-		local computed = Computed(scope, function(use)
+		local computed = Computed(scope, function(use, scope)
 			return use(5)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		local computed = Computed(scope, function(use)
```

Reply:
reverted